### PR TITLE
docs(progress): add dynamic presentation

### PR DESCRIPTION
**Pull request by YunYouJun**
Add dynamic presentation for [progress](https://element-plus.org/#/zh-CN/component/progress), and format doc table.

For example:

https://user-images.githubusercontent.com/25154432/121490334-7c10f500-ca07-11eb-8a7f-ccab74f954ea.mov

https://user-images.githubusercontent.com/25154432/121490347-7f0be580-ca07-11eb-9b62-c0b73e5edc16.mov

---

Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by JeremyWuuuuu** [comment]
Same error as before. You've only changed the js code but you did not modify the templates which means nothing can be seen on the screen except for `zh-CN`, you also need to modify the template in other language packs.

**Comment by YunYouJun** [comment]
Oops. Done.

File: website/docs/en-US/progress.md [hunk 1]
```diff
@@ -93,6 +95,11 @@ You can use `color` attr to set the progress bar color. it accepts color string,
           this.percentage = 0;
         }
       }
+    },
+    mounted() {
+      setInterval(() => {
```
Comment (sheepsheepsheep-tiaobudong) [inline website/docs/en-US/progress.md:100]: Can we use requestAnimationFrame instead?

**Review by kooriookami — APPROVED** [review]

File: website/docs/en-US/progress.md [hunk 2]
```diff
@@ -153,6 +162,11 @@ You also can specify `type` attribute to `dashboard` to use dashboard progress b
...
```
Comment (HerringtonDarkholme) [inline website/docs/en-US/progress.md:167]: Can we use `requestAnimationFrame`?

**Pull request merged**